assets/css/inloop.css

```
@@ -72,6 +72,18 @@ code {
   color: #ddd;
 }
 
+/*
+ * Insert an invisible spacer on selected headings
+ * to force them below the sticky tab bar
+ */
+.static-tab-content :target:before {
```

```suggestion
.static-tab-content :target::before {
```

According to https://developer.mozilla.org/en-US/docs/Web/CSS/::before I think we should use `::`. Developer tools display it like this as well, and it makes sense because in front of `::` no space is allowed, but in front of `:` it is :-)